### DISCUSSION TITLE
[#1040906] In-game/offline documentation

### DISCUSSION THREAD

**Discussion by SmokeyStack** [Mojang/minecraft-editor#41] [Feedback]
Similar to how add-ons have a html doc shipped with the vanilla packs, Editor should have an offline mode. When clicking on the Quick Start or Documentation button, it should open a menu similar to the How To Play screen in normal Minecraft. In the screen a link should also be provided to direct to the online version.

**Comment by minecraftchloe** [comment]
This is a fantastic idea!
